### PR TITLE
Support cloud regions with multiple clusters in the docs

### PR DESCRIPTION
**Pull request by sanderson**
This PR updates the existing InfluxDB Cloud URL functionality to support regions with multiple clusters.

_**(Don't merge until the new AWS us-west-2-2 region is live)**_

**Updated modal**
image

**Updated Cloud regions page**
image

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/regions.md [hunk 1]
```diff
@@ -19,4 +19,16 @@ Use the URLs below to interact with your InfluxDB Cloud instances with the
 
 <a href="https://www.influxdata.com/influxdb-cloud-2-0-provider-region/" target="_blank" class="btn">Request a cloud region</a>
 
+{{% note %}}
+#### Regions with multiple clusters
+Some InfluxDB Cloud regions have multiple InfluxDB clusters, each with a unique URL.
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/regions.md:24]:
```suggestion
Some InfluxDB Cloud regions have multiple Cloud clusters, each with a unique URL.
```

File: content/influxdb/cloud/reference/regions.md [hunk 2]
```diff
@@ -19,4 +19,16 @@ Use the URLs below to interact with your InfluxDB Cloud instances with the
...
+To find out which cluster you're using, [login to your InfluxDB Cloud organization](https://cloud2.influxdata.com)
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/regions.md:25]:
```suggestion
To find your cluster URL, [log in to your InfluxDB Cloud organization](https://cloud2.influxdata.com)
```

File: content/influxdb/cloud/reference/regions.md [hunk 3]
```diff
@@ -19,4 +19,16 @@ Use the URLs below to interact with your InfluxDB Cloud instances with the
...
+and look at the URL of your organization. The first subdomain identifies your
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/regions.md:26]:
```suggestion
and review your organization URL. The first subdomain identifies your 
```

**Review by kelseiv — APPROVED** [review]

**Review by jstirnaman — APPROVED** [review]

**Review by gaughen — APPROVED** [review]
Reviewed the pretty modals and those match what came out of our discussion, and look good. Thank you!

**Comment by barbaranelson** [comment]
Looks good to me.  Thanks.

**Review by barbaranelson — APPROVED** [review]

File: content/influxdb/cloud/reference/regions.md [hunk 4]
```diff
@@ -19,4 +19,16 @@ Use the URLs below to interact with your InfluxDB Cloud instances with the
...
 
+{{% note %}}
+#### Regions with multiple clusters
+Some InfluxDB Cloud regions have multiple InfluxDB clusters, each with a unique URL.
+To find out which cluster you're using, [login to your InfluxDB Cloud organization](https://cloud2.influxdata.com)
+and look at the URL of your organization. The first subdomain identifies your 
+InfluxDB Cloud cluster. For example:
+
+<pre class="highlight">
+https://<span class="bp" style="font-weight:bold;margin:0 .15rem">us-west-2-1</span>.aws.cloud2.influxdata.com/orgs/03a2bbf46249a000/...
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/cloud/reference/regions.md:30]: That's something we could possibly implement. I've only done this in this and one other spot, so I've kept it a manual process. Do you have other use cases for it?

**Pull request merged**